ail_vru_uc1_scenario/docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
     volumes:
       - /tmp/.X11-unix:/tmp/.X11-unix
       - /opt/carma-simulation/logs:/opt/carma-simulation/logs
+      - ./carla-recorder/:/app/scenario_runner/metrics/data/
```

Comment:
Why are we mounting this volume?

Reply:
The scenario runner image creates a criteria json file and the carla server running in the CDA Sim container creates a carla recording. Using the criteria json file we can parse the log file with the metrics module in carla-scenario-runner to return information about metrics.

Reply:
Behind the scenes when I specify --record <path> carla-scenario-runner calls carla-client.start_recording(<SCENARIO_RUNNER_ROOT>/<path>) and stop recording once the scenario completes. This prompts the carla server to create a recording file under this path.